src/loki/height_action.cc

```diff
@@ -28,7 +28,7 @@ std::vector<PointLL> loki_worker_t::init_height(Api& request) {
 
   // resample the shape
   bool resampled = false;
-  if (options.has_resample_distance()) {
+  if (options.has_resample_distance_case()) {
```

Comment:
and this is what the changes in code look like. i named all of the `oneof` with the `has_` prefix so that the beginnign wouldnt change, but there was no easy way to avoid having to put the `_case` on the end because the damned bindings that protoc creates make the normal `has_field` method `private` grr